src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -0,0 +1,62 @@
+package com.axibase.tsd.client;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+import java.io.IOException;
+import java.io.Serializable;
+import java.net.Socket;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+@Slf4j
+public class SelfSignedTrustManager implements X509TrustManager, Serializable, Cloneable {
+    private final static  String CONSTANT = "message";
+
+    public SelfSignedTrustManager() throws NoSuchAlgorithmException, KeyStoreException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Non derivable method new com.axibase.tsd.client.SelfSignedTrustManager() declares throwing an exception that isn't thrown [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=fb-contrib%3ABED_BOGUS_EXCEPTION_DECLARATION)